nameko_sqlalchemy/pytest_fixtures.py

```diff
@@ -36,6 +36,24 @@ def db_url(request):
 
 
 @pytest.fixture(scope='session')
+def create_engine_kwargs():
```

Comment:
How do you feel about calling this `engine_options`?

Precisely, they are kwargs passed to `create_engine`, but I think in reality they are options for constructing the engine.